src/core1_task.rs

```diff
@@ -966,6 +1129,19 @@ pub fn core_1_task(
                             logged_pi_powered_down = true;
                         }
 
+                        //do this when powered down so preview stays up
```

Comment:
I don't understand this comment about preview staying up when powered down.

Reply:
adjusted comment, but idea is whenever the pi is on always be in thermal mode when possible, so that the thermal preview is always shown. And only reboot into audio mode when pi is powered down.

It's probably very unlikely that we somehow miss an audio alarm from firing before the rp2040 goes to sleep. but have the logic to handle this in audio mode, so prefer to boot into there and let the audio mode logic handle everything.